### PR TITLE
chore:update apklib version

### PR DESCRIPTION
**Pull request by tonaim**
updated libapk version to libapk.so.2.14.0, as in alpine 3.18 libapk.so.3.12.0 is not available. also updated the application.cfg files with 3.17 and 3.18 alpine versions.

### CODE REVIEW

File: secfixes_tracker/version.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 VersionFuzzy = 8
 
 
-libapk = cdll.LoadLibrary('libapk.so.3.12.0')
+libapk = cdll.LoadLibrary('libapk.so.2.14.0')
```
Comment (knqyf263) [inline secfixes_tracker/version.py:11]: downgrade?

**Review by knqyf263 — COMMENTED** [review]
Looks good, but I'm wondering why we need to downgrade the libapk version. Not update?

File: secfixes_tracker/version.py (hunk 1 above)
Comment (tonaim) [reply secfixes_tracker/version.py:11]: until alpine 3.17, /lib/libapk.so.3.12.0 is available, from alpine 3.18 I can see only lib/libapk.so.2.14.0.

cron job uses alpine:latest.

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**